test/pseudo-tty/test-tty-isatty.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('../common');
+const { strictEqual } = require('assert');
+const { isatty } = require('tty');
+
+strictEqual(isatty(0), true, 'stdin is not a tty');
+strictEqual(isatty(1), true, 'stdout is not a tty');
+strictEqual(isatty(2), true, 'stderr is not a tty');
+
+strictEqual(isatty(-1), false, 'number that isn\'t a tty, is');
+strictEqual(isatty(5), false, 'number that isn\'t a tty, is');
+strictEqual(isatty(1.1), false, 'number that isn\'t a tty, is');
```

Comment:
The messages here are somewhat confusing out of my perspective. Maybe write something like e.g.
`-1 reported to be a tty but should not`?

Reply:
Yeah I'll clarify these.